### PR TITLE
docs: update `pytorch_lightning` imports

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

unify references and imports in examples to be `lightning.`

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

### CODE REVIEW

File: docs/source-pytorch/advanced/training_tricks.rst [hunk 1]
```diff
@@ -429,4 +429,4 @@ For this, all data pre-loading should be done on the main process inside :meth:`
     trainer = Trainer(accelerator="gpu", devices=2, strategy="ddp_spawn")
     trainer.fit(model, datamodule)
 
-See the `graph-level <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pytorch_lightning/gin.py>`_ and `node-level <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pytorch_lightning/graph_sage.py>`_ prediction examples in PyTorch Geometric for practical use-cases.
+See the `graph-level <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/lightning.pytorch/gin.py>`_ and `node-level <https://github.com/pyg-team/pytorch_geometric/blob/master/examples/lightning.pytorch/graph_sage.py>`_ prediction examples in PyTorch Geometric for practical use-cases.
```
Comment (awaelchli) [inline docs/source-pytorch/advanced/training_tricks.rst:432]: accidental find and replace
maybe double check other links too if there were any

File: docs/source-pytorch/ecosystem/community_examples.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Community Examples
 - `NeuralTexture (CVPR) <https://github.com/henzler/neuraltexture>`_
 - `Recurrent Attentive Neural Process <https://github.com/3springs/attentive-neural-processes>`_
 - `Siamese Nets for One-shot Image Recognition <https://github.com/bhiziroglu/Siamese-Neural-Networks>`_
-- `Speech Transformers <https://github.com/tongjinle123/speech-transformer-pytorch_lightning>`_
+- `Speech Transformers <https://github.com/tongjinle123/speech-transformer-lightning.pytorch>`_
```
Comment (awaelchli) [inline docs/source-pytorch/ecosystem/community_examples.rst:18]: here too

File: docs/source-pytorch/ecosystem/community_examples.rst [hunk 3]
```diff
@@ -32,5 +32,5 @@ Community Examples
 PyTorch Ecosystem Examples
 ==========================
 
-- `PyTorch Geometric: Deep learning on graphs and other irregular structures <https://github.com/rusty1s/pytorch_geometric/tree/master/examples/pytorch_lightning>`_.
-- `TorchIO, MONAI and Lightning for 3D medical image segmentation <https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_MONAI_PyTorch_Lightning.ipynb>`_.
+- `PyTorch Geometric: Deep learning on graphs and other irregular structures <https://github.com/rusty1s/pytorch_geometric/tree/master/examples/lightning.pytorch>`_.
+- `TorchIO, MONAI and Lightning for 3D medical image segmentation <https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_MONAI_lightning.pytorch.ipynb>`_.
```
Comment (awaelchli) [inline docs/source-pytorch/ecosystem/community_examples.rst:36]: here too

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]
Can you check the references in the docstrings too in a follow-up?

**Comment by Borda** [comment]
> Can you check the references in the docstrings too in a follow-up?

they were already done in #16594 but checked again and there was just one missing [2c97cda](https://github.com/Lightning-AI/lightning/pull/16864/commits/2c97cdadebad739d53b00a956e501afaf91f9156)

**Review by carmocca — COMMENTED** [review]
The "pytorch" docs fail to build because they import `lightning` but the app requirements are not installed.

Since the docs will be merged, should the docs jobs be merged too?

**Comment by Borda** [comment]
This need to be picked for building 1.9 docs as RTFD does not allow define env. variable in yaml file...
